concordium-node/src/p2p/peers.rs

```diff
@@ -54,17 +55,17 @@ impl P2PNode {
     /// Measures the node's average byte throughput as bps i.e., bytes per
     /// second.
     pub fn measure_throughput(&self, peer_stats: &[PeerStats]) -> anyhow::Result<()> {
-        let prev_bytes_received = self.stats.received_bytes.get();
-        let prev_bytes_sent = self.stats.sent_bytes.get();
+        let prev_bytes_received = self.stats.last_throughput_measurement_received_bytes.get();
+        let prev_bytes_sent = self.stats.last_throughput_measurement_sent_bytes.get();
 
         let (bytes_received, bytes_sent) = peer_stats
             .iter()
             .filter(|ps| ps.peer_type == PeerType::Node)
             .map(|ps| (ps.bytes_received, ps.bytes_sent))
             .fold((0, 0), |(acc_i, acc_o), (i, o)| (acc_i + i, acc_o + o));
 
-        self.stats.received_bytes.set(bytes_received);
-        self.stats.sent_bytes.set(bytes_sent);
+        self.stats.last_throughput_measurement_received_bytes.set(bytes_received);
+        self.stats.last_throughput_measurement_sent_bytes.set(bytes_sent);
 
         let now = Utc::now().timestamp_millis();
         let (avg_bps_in, avg_bps_out) = calculate_average_throughput(
```

Comment:
Why is this avg still computed using the bytes_received and bytes_sent? And not using the new totals?

Reply:
I have changed this

Reply:
Am I missing something? It does not seem you have changed this.

Reply:
Weird, Github might be acting up or something, I have pushed another commit. Everything is up to date according to git when I push,

Reply:
Ammending and force pushing seems to have fixed it